redbeat/schedulers.py

```diff
@@ -246,7 +253,7 @@ def due_at(self):
 
     @property
     def key(self):
-        return self.app.redbeat_conf.key_prefix + self.name
+        return self.app.redbeat_conf.key_prefix + str(self.name)
```

Comment:
Why ```str()``` here? It seems unrelated to the PR.

Reply:
Ah, sorry, yeah, i forgot to mention that I kept getting type errors in this line regarding bytes vs strings here.  I narrowed it down to name being in bytes instead of strings, which fixed the issue.  I tried to trace where `name` was being set, but had a hard time figuring out.  If you know where that is, I could investigate further and find out why it's being set as bytes instead of strings for my usecase.

Reply:
name is either coming from [configuration](https://github.com/sibson/redbeat/blob/master/redbeat/schedulers.py#L365) or [existing in redis](https://github.com/sibson/redbeat/blob/master/redbeat/schedulers.py#L401).  Or it  could be creating Entries programmatically somewhere in your code.   If there is a problem with RedBeat handling name I'd like to see that fixed in a separate PR.

Reply:
Thanks for the leads, I'll investigate that separately and open another PR when I find the culprit.  Thanks!

Reply:
Removed this change from the latest update 👍